src/app/qgsvectorlayerproperties.h

```diff
@@ -195,6 +195,7 @@ class APP_EXPORT QgsVectorLayerProperties : public QgsOptionsDialogBase, private
     QMenu *mLoadStyleMenu = nullptr;
 
     QAction *mActionLoadStyle = nullptr;
+    QAction *mActionSaveStyle = nullptr;
```

Comment:
`mActionSaveStyleAs` at next line should be used instead

Reply:
Just removed the mActionSaveStyleAs declaration as this wasn't being used anywhere